internal/cluster/transition.go

```diff
@@ -167,6 +190,17 @@ func (th *transitionHandler) updateTransitionCluster(log logrus.FieldLogger, db
 	}
 }
 
+func (th *transitionHandler) updateClusterInstallProgress(log logrus.FieldLogger, db *gorm.DB, state *stateCluster,
```

Comment:
isn't this function is the same as `updateTransitionCluster` why do you need another?

Reply:
this function calls `updateClusterProgress` while `updateTransitionCluster` calls `updateClusterStatus`

Reply:
But this function don't do any logic and just call to `updateClusterProgress` why not call it directly?

Reply:
ok, now calling `updateClusterProgress` directly